OpenRiaServices.M2M.Demo.Web/OpenRiaServices.M2M.Demo.Web.csproj

```diff
@@ -53,17 +53,17 @@
     <Reference Include="EntityFramework.SqlServer">
       <HintPath>..\packages\EntityFramework.6.1.0\lib\net40\EntityFramework.SqlServer.dll</HintPath>
     </Reference>
-    <Reference Include="OpenRiaServices.DomainServices.EntityFramework, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="OpenRiaServices.EntityFramework, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\OpenRiaServices.EntityFramework.4.4.0.0-Beta1\lib\net40\OpenRiaServices.DomainServices.EntityFramework.dll</HintPath>
+      <HintPath>..\packages\OpenRiaServices.EntityFramework.4.4.0.0-Beta1\lib\net40\OpenRiaServices.EntityFramework.dll</HintPath>
     </Reference>
-    <Reference Include="OpenRiaServices.DomainServices.Hosting, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="OpenRiaServices.Hosting, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\OpenRiaServices.Server.4.4.0.0-Beta1\lib\OpenRiaServices.DomainServices.Hosting.dll</HintPath>
+      <HintPath>..\packages\OpenRiaServices.Server.4.4.0.0-Beta1\lib\OpenRiaServices.Hosting.dll</HintPath>
     </Reference>
-    <Reference Include="OpenRiaServices.DomainServices.Server, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+    <Reference Include="OpenRiaServices.Server, Version=4.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
The paths seems off, they should either refer to the old v4 library with old path or be updated to rc1 with new paths. The same applies to all changes to demo projects

If you plan to do an immediate follow up pr which fixes the demo then I am fine with completing as is and then fixing the demo